ci-operator/config/openshift/console-operator/openshift-console-operator-master.yaml

```diff
@@ -89,6 +89,10 @@ tests:
         requests:
           cpu: 100m
     workflow: ipi-aws
+- as: e2e-aws-single-node
```

Comment:
Should this go in openshift-console-operator-release-4.8.yaml and openshift-console-operator-release-4.9.yaml, too?

Reply:
I think this happens automatically after a merge. I'll verify and get back to you

Reply:
It seems to have happened in other repositories where we added this job. Let's try to merge - if it doesn't happen we'll add it in a separate PR